torch/quantization/_quantize_script.py

```diff
@@ -81,8 +81,7 @@ def convert_script(model, inplace=False):
     if not inplace:
         model = model.copy()
     model = wrap_cpp_module(torch._C._jit_pass_insert_quant_dequant(model._c, 'forward', False))
-    if 'fbgemm' in torch.backends.quantized.supported_engines:
-        torch._C._jit_pass_insert_prepack_unpack(model._c)
+    model = wrap_cpp_module(torch._C._jit_pass_quant_finalize(model._c))
```

Comment:
Also, can you add tests for convert_script with and without debug=True to ensure that this API is tested?

Reply:
tests are added in later PR: https://github.com/pytorch/pytorch/pull/33895/files

Reply:
For completeness, would it be possible to move them into this PR?